README.md

```diff
@@ -194,7 +194,7 @@ data-validation (--verbose or -v) (--log-level or -ll) validate row
                         If target filter is not provided, the source filter will run on source and target tables.
                         See: *Filters* section
   [--config-file or -c CONFIG_FILE]
-                        YAML Config File Path to be used for storing validations and other features.
+                        YAML Config File Path to be used for storing validations and other features. Supports GCS and local paths.
                         See: *Running DVT with YAML Configuration Files* section
   [--config-file-json or -cj CONFIG_FILE_JSON]
```

Comment:
This is probably unrelated to your changes but noting it anyway. If I create a json config file, e.g.:
```
data-validation validate row -sc ora -tc pg \
-tbls=s.c --primary-keys id --hash '*' \
--filter-status fail -cj /tmp/c.json
```

I cannot validate with it:
```
data-validation configs run -cj /tmp/c.json
...
data-validation: error: unrecognized arguments: /tmp/c.json
```

Or am I using it incorrectly?

Reply:
Yeah, we only support saving to JSON, not running JSON through the CLI. JSON should only be used for Cloud Run or other application based use cases as a request body